PRS5001-6000/PRS5276Helena.xml

```diff
@@ -55,7 +55,6 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                <occupation type="political">empress</occupation>
                <birth notBefore="0248" notAfter="0250">248-250</birth>
                <death when="0328">328</death>
-               <nationality type="ByzantineEmpire"></nationality>
                <floruit>Mother of emperor Constantine. In Ethiopian tradition she features in various texts related to
                the finding of the Holy Cross.</floruit>
```

Comment:
I would annotate Holy Cross `<ref type="authFile" corresp="TrueCross">Holy Cross</ref>`

Reply:
Great idea, I didn't know it was possible